index.html

```diff
@@ -31,9 +33,42 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>
+            <div id="header">
+                <h1>Nicholas McVickar</h1>
+            </div>
+            <div id="content">
+                <h2>"A bird does not sing because it has an answer, it sings because it has a song."</h2> <br>
+                <h2>-Joan Walsh Anglund</h2>
+            </div>
+            <div id="footer">
+                <img style="width:80px; height: 70px;" src="https://lh3.googleusercontent.com/rDWANKL8svn89tlBt7IiZPdi3LTxeoEBoNOaOnoYnOeMWloYNIqbE7ab8pd1Msb1ll7aq2eQ5AGtHYNFD-Lki4_S9MMT7YP_MNbW8MUZHNCJSI3H082VEiuzdeXWr7dIqR4aC3NOR11PEvIj4j46z3meHNdu7RlABvADiJR7i6vw-e9rctcmhoYmZ8z-x0UYLREe7j4xr6U8LBWpNg81I20gyBiRudCN-1AizqnkgsD-6rCwWh5UM6V8zF0QYXL7MPDvvp4f1I93JQ9s127EuFj6pKFNKLi8FEifUUc2ckm2oCK6CB51xIiXPb-6E0Iq3-KAbkRUXrfOni4EXKL2KNeynrhFGUBxfP2P4eCexjrnHsQrKJxC_9V2v3J2q5acvw6cSzyL7sK0eY-DJJyKnA5vn3qx5HNX9FnhCxevQKomRmiAEQ4oGeElTmFEYAD3ISrIC2DUNocayLsHwk9z0fi4GqXguxYUCrDAAds4U94fd6xuP0v20O8oWxPo4ZucsT0-5zh0ZCM91T2m0h0_9n7RiqliByRTTTeN6KxwtEUenKM8gFA6a-3BgmGheq-Ekta_0RMrMFDvPt8w33XgwqE6WoLhU4zGc_Pj_lcW=w488-h428-no"
+                    alt="My Face">
+                <h3>
+                    <ul>
+                        <li>
+                            <a href="tel:+16187999959">
+                                Give me a call
+                            </a>
+                        </li>
+                        <li>
+                            <a href="mailto:nmcvickar@aol.com">
+                                <strong>Email me</strong>
+                            </a>
+                        </li>
+                        <li>
+                            My <a href="https://github.com/NMcVickar" target="_blank">
+                                <strong>github</strong>
```

Comment:
We should favor `font-weight` over the `<strong>` tag generally (and in this case specifically)